src/main/java/com/dummyblog/app/repository/entity/User.java

```diff
@@ -0,0 +1,160 @@
+package com.dummyblog.app.repository.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Id
+	private Long id;
+	
+	@Column(nullable = false)
+	private String name;
+	
+	@Column(nullable = false)
+	private String password;
+	
+	@Column(nullable = false)
+	private Integer age;
+	
+	@Column(unique = true, nullable = false)
+	private String email;
+	
+	@Enumerated(EnumType.STRING)
+	private Privileges role;
+	
+	public User() {
+		
+	}
+
+	public User(String name, String email, String password, Privileges role) {
```

Comment:
Mnior: amikor túlterheled a konstruktort, akkor a kód duplikáció elkerülése érdekben nyugodtan hívhatod a this kulcsszóval a bővebb konstruktort, a hiányzó paramétereknél meg simán null referenciát adsz át